pkg/apis/servicecatalog/v1alpha1/doc.go

```diff
@@ -19,5 +19,7 @@ limitations under the License.
 // +k8s:openapi-gen=true
 // +k8s:defaulter-gen=TypeMeta
 
+// Package v1alpha1 defines all of the versioned (v1alpha1) definitions
+// of the service catalog model.
```

Comment:
hmmm, I'm not sure. Wouldn't v1alpha1 still appear in the API version string?  @pmorie ??